SuperChamps/contracts/Token/SuperChampsToken.sol

```diff
@@ -25,13 +25,15 @@ contract SuperChampsToken is ERC20, ERC20Permit {
     modifier isAdminOrUnlocked() {
         require(!transfersLocked || 
                 permissions.hasRole(IPermissionsManager.Role.TRANSFER_ADMIN, _msgSender()) ||
-                permissions.hasRole(IPermissionsManager.Role.TRANSFER_ADMIN, tx.origin));
+                permissions.hasRole(IPermissionsManager.Role.TRANSFER_ADMIN, tx.origin),
```

Comment:
will these changes have ant effect as token contract is already deployed?

Reply:
This minor change just never got merged in.